README.md

```diff
@@ -7,80 +7,62 @@
-
-Further information is included in the following documents:
-
-- [Oak Development](docs/development.md) covers practical steps for getting a
-  development Oak system up and running.
-- [Oak Functions](oak_functions/README.md) presents our computing platform for
-  developing stateless applications in a privacy preserving way.
+To do so, Oak relies on running _Oak Trusted Enclave Applications_ in a
+[Trusted Execution Environment (TEE)](https://en.wikipedia.org/wiki/Trusted_execution_environment).
+An example of a Trusted Enclave Application is
+[Oak Functions](/oak_functions/README.md). The Trusted Enclave Application can
+provide the client cryptographically attested evidence of the security
+properties by [Remote Attestation](./docs/remote-attestation.md). Together with
+[Transparent Release](https://github.com/project-oak/transparent-release) this
+binds the open-source source code to the remotely attested binary running inside
+the TEE. In order to feasibly review all the source code running inside the TEE,
+and minimize our trusted computing base, Oak provides the following
+infrastructure: [stage 0](/stage0/),
+[Oak Restricted Kernel](/oak_restricted_kernel/) and a controlled communications
+interfaces---the [Oak Comms Channel](/oak_channel/) and [microRPC](/micro_rpc/).
 
 ## Parties involved
 
-- **Oak Trusted Runtime Authors**: The authors of the code in this repository;
+- **Trusted Enclave Application Authors**: The authors writing the Trusted
+  Enclave Application running on Oak.
+- **Oak Infrastructure Authors**: The authors of the code in this repository;
   mostly this corresponds to the Project Oak team, but also any contributors,
   and, by extension, the authors of third party dependencies used in Oak.
-- **TEE Manufacturer**: The entity in charge of manufacturing the CPU or System
-  on a Chip (SoC) supporting the TEE, including hardware, software, and
-  cryptographic keys.
 - **Platform Provider**: The entity in charge of maintaining and running the
   combined hardware and software stack surrounding the TEE, for instance a cloud
   provider; this includes their software, hardware, and employees.
-- **Untrusted Business Logic Authors**: The authors of the untrusted business
-  logic, compiled to Wasm, that is executed by the Oak Functions Runtime.
+- **TEE Manufacturer**: The entity in charge of manufacturing the TEE, including
+  hardware, software, and cryptographic keys.
 
 ## Threat Model
 
 - **untrusted**:
   - most hardware (memory, disk, motherboard, network card, external devices)
-  - Operating System (kernel, drivers, libraries, applications)
   - Platform Provider
-  - Untrusted Business Logic Authors
+  - Host Operating System (kernel, drivers, libraries, applications)
+  - Hypervisor / VMM
 - **trusted-but-verifiable**:
-  - Oak Trusted Runtime Authors (their actions are verifiable via
+  - Oak Infrastructure Authors
+  - Trusted Enclave Application Authors
+  - (their actions are verifiable via
```

Comment:
This should be nested under the previous one, I think

Reply:
Rephrased to "trusted-but-transparent". Should the "Trusted Enclave Application Authors" be on there? I think so. Other opinions?